### ISSUE TITLE
A newly created set of plastic flaps creates atmosphere traps where it was spawned in.

### ISSUE THREAD

**Issue by necromanceranne** [tgstation/tgstation#71957]
## Reproduction:

Step 1) Take a stack of plastic

Step 2) Construct some airtight plastic flaps. Do NOT tamper with them.

Step 3) Create, say, a fire underneath the flaps using lit plasma.

Step 4) remove the plastic flaps.

Congratulations, that atmosphere is now unable to move or have gas flow in. This happens even when you move the flaps away for some reason (or they just melt from fire)

![very interesting](https://user-images.githubusercontent.com/40847847/207001658-1b862357-f98a-48f3-afb7-f248d37c1226.png)

Now, it is important to note that this does not work with any plastic flaps that have been screwdrivered up, moved, or anything of that sort. Only ones spawned in via construction, mapload, etc. Even if you put it back down, the flaps no longer block atmosphere like this.

**Comment by Serxule** [comment]
i mean yeah plastic flaps are airtight so it makes sense

**Comment by necromanceranne** [comment]
> i mean yeah plastic flaps are airtight so it makes sense

No, see, it seals it in even when the plastic flaps literally do not exist in that tile anymore.

**Comment by Sealed101** [comment]
lol that wasnt fixed
